### PR TITLE
Add tools for security files generation

### PR DESCRIPTION
**Pull request by mikaelarguedas**
addresses https://github.com/ros2/ros2/pull/340#discussion_r119451953

connects to ros2/ros2#339

### CODE REVIEW

**Comment by dirk-thomas** [comment]
Would it make sense to refactor this into a command of the generic `ros2` command line tools?

**Comment by mikaelarguedas** [comment]
After discussing it offline we decided to merge this and once the ros2cli PRs land we will evaluate if they should be integrated in the generic ros2 commandline tool and if yes if we do it before or after beta2

**Comment by mikaelarguedas** [comment]
I updated the Dockerfile and the Readme to leverage the code that made it to master so far. Once this is merged and a few other PRs I'll open a follow-up PR to update the Dockerfile accordingly.

Ready for review (and testing ?)

File: sros2.repos [hunk 1]
```diff
@@ -0,0 +1,161 @@
+repositories:
```
Comment (wjwwood) [inline sros2.repos]: @mikaelarguedas should this be synced with the upstream? This is the current diff:

```diff
--- repositories: (Selection)
+++ (clipboard)
@@ -51,10 +51,6 @@
     type: git
     url: https://github.com/ros2/ament_cmake_ros.git
     version: master
-  ros2/cli_tools:
-    type: git
-    url: https://github.com/ros2/cli_tools.git
-    version: master
   ros2/common_interfaces:
     type: git
     url: https://github.com/ros2/common_interfaces.git
@@ -90,11 +86,15 @@
   ros2/rcl:
     type: git
     url: https://github.com/ros2/rcl.git
-    version: sros2
+    version: master
   ros2/rcl_interfaces:
     type: git
     url: https://github.com/ros2/rcl_interfaces.git
     version: master
+#  ros2/rclc:
+#    type: git
+#    url: https://github.com/ros2/rclc.git
+#    version: master
   ros2/rclcpp:
     type: git
     url: https://github.com/ros2/rclcpp.git
@@ -107,18 +107,46 @@
     type: git
     url: https://github.com/ros2/rcutils.git
     version: master
+  ros2/realtime_support:
+    type: git
+    url: https://github.com/ros2/realtime_support.git
+    version: master
   ros2/rmw:
     type: git
     url: https://github.com/ros2/rmw.git
-    version: sros2
+    version: master
+  ros2/rmw_connext:
+    type: git
+    url: https://github.com/ros2/rmw_connext.git
+    version: master
   ros2/rmw_fastrtps:
     type: git
     url: https://github.com/ros2/rmw_fastrtps.git
-    version: sros2
+    version: master
   ros2/rmw_implementation:
     type: git
     url: https://github.com/ros2/rmw_implementation.git
-    version: sros2
+    version: master
+  ros2/rmw_opensplice:
+    type: git
+    url: https://github.com/ros2/rmw_opensplice.git
+    version: master
+  ros2/robot_model:
+    type: git
+    url: https://github.com/ros2/robot_model.git
+    version: ros2
+  ros2/robot_state_publisher:
+    type: git
+    url: https://github.com/ros2/robot_state_publisher.git
+    version: ros2
+  ros2/ros1_bridge:
+    type: git
+    url: https://github.com/ros2/ros1_bridge.git
+    version: master
+  ros2/ros2cli:
+    type: git
+    url: https://github.com/ros2/ros2cli.git
+    version: master
   ros2/rosidl:
     type: git
     url: https://github.com/ros2/rosidl.git
@@ -131,18 +159,26 @@
     type: git
     url: https://github.com/ros2/rosidl_typesupport.git
     version: master
+  ros2/sros2:
+    type: git
+    url: https://github.com/ros2/sros2.git
+    version: master
   ros2/system_tests:
     type: git
     url: https://github.com/ros2/system_tests.git
-    version: master
-  ros2/sros2:
-    type: git
-    url: https://github.com/ros2/sros2.git
     version: master
   ros2/tinyxml_vendor:
     type: git
     url: https://github.com/ros2/tinyxml_vendor.git
     version: master
+  ros2/tlsf:
+    type: git
+    url: https://github.com/ros2/tlsf.git
+    version: master
+#  ros2/tutorials:
+#    type: git
+#    url: https://github.com/ros2/tutorials.git
+#    version: master
   ros2/urdfdom:
     type: git
     url: https://github.com/ros2/urdfdom.git
```

I don't know enough about this to say if it is so.

Comment (mikaelarguedas) [inline sros2.repos]: Actually the [update from yesterday](https://github.com/ros2/sros2/pull/3/commits/3a4d7dbe7e9c893aff97523492f546b9d11ace9a) should allow to remove this file completely. Looks like I forgot to commit the deletion and updated only the Dockerfile but not the other sections of the README. Thanks for pointing it out !

**Comment by mikaelarguedas** [comment]
thanks @wjwwood and @Kukanani for reviewing and testing.

Merging now

**Pull request merged**